packages/expo-cli/src/commands/build-native/iOSBuilder.ts

```diff
@@ -67,6 +71,60 @@ class iOSBuilder implements Builder {
     this.credentials = await provider.getCredentialsAsync(credentialsSource);
   }
 
+  public async configureProjectAsync(): Promise<void> {
+    // TODO: add simulator flow
+    // assuming we're building for app store
+    if (!this.credentials) {
+      throw new Error('Call ensureCredentialsAsync first!');
+    }
+    const bundleIdentifier = await getBundleIdentifier(this.ctx);
+
+    const spinner = ora('Making sure your iOS project is set up properly');
+
+    const profileName = ProvisioningProfileUtils.readProfileName(
+      this.credentials.provisioningProfile
+    );
+    const appleTeam = ProvisioningProfileUtils.readAppleTeam(this.credentials.provisioningProfile);
+
+    const { projectDir } = this.ctx;
+    IOSConfig.BundleIdenitifer.setBundleIdentifierForPbxproj(projectDir, bundleIdentifier, false);
+    IOSConfig.ProvisioningProfile.setProvisioningProfileForPbxproj(projectDir, {
+      profileName,
+      appleTeamId: appleTeam.teamId,
+    });
+
+    try {
+      await gitUtils.ensureGitStatusIsCleanAsync();
+      spinner.succeed();
+    } catch (err) {
+      if (/Please commit all changes/.exec(err.message)) {
```

Comment:
Maybe instead of relying on `err.message`, adding an error code would be better?